presto/inference.py

```diff
@@ -255,6 +243,34 @@ def process_parquet(df: pd.DataFrame) -> pd.DataFrame:
         "valid_date",
     ]
 
+    # add dummy value + rename stuff for compatibility with existing functions
+    df["OPTICAL-B8A"] = 65535
+
+    rename_dict = {
```

Comment:
This `process_parquet` method needs to be checked again. Though on the one hand that this one was taken out, and other hand if it's still in, normally things like this renaming were no longer necessary as it was already taken care of. TBD.